server/utils/constants.ts

```diff
@@ -1,14 +1,40 @@
-import { DefaultHeaders, SecurityAnalyticsApi } from "../models/interfaces";
```

Comment:
Not blocking for this draft, but something that I also need to be mindful of is that we need to add the license header at the top of our files.
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

Reply:
added locally will be pushed in next PR